modules/social_features/social_event/asyncapi.yml

```diff
@@ -0,0 +1,136 @@
+channels:
+  eventCreate:
+    address: com.getopensocial.cms.event.create
+    messages:
+      eventCreate:
+        payload:
+          allOf:
+            - $ref: '#/components/schemas/cloudEventsSchema'
+            - type: object
+              properties:
+                data:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                      description: The UUID of the event.
+                    created:
+                      type: string
+                      format: date-time
+                      description: The creation time of the event.
+                    updated:
+                      type: string
+                      format: date-time
+                      description: The last updated time of the event.
+                    status:
+                      type: string
+                      description: The status of the event.
+                    label:
+                      type: string
```

Comment:
This is currently a string, but in fact we can have but two options only. Is there a way to define that or is that not possible @tregismoreira?

Reply:
Wrong field. I meant `status`, sorry!

Reply:
In the final documentation it'll not be clear that there is only `published` and `unpublished`. Just that they will receive a `string`.

If there is a way to define the `published` and `unpublished` that would be best imho.

Reply:
I added a property `enum` with those two options.